src/mscorlib/src/System/LowLevelConsole.cs

```diff
@@ -10,15 +10,15 @@
 namespace System
 {
     //
-    // Simple limited console class for internal printf-style debugging in mscorlib
-    // and low-level tests that just want to depend on mscorlib.
+    // Simple limited console class for internal printf-style debugging in System.Private.CoreLib
+    // and low-level internal tests that want to call System.Private.CoreLib directly
```

Comment:
Answers my question.  Does the implementation work on Unix?  I guess it does via the coreclr pal?